keras_hub/src/models/sam/sam_image_segmenter_preprocessor.py

```diff
@@ -1,12 +1,23 @@
+import keras
+
 from keras_hub.src.api_export import keras_hub_export
 from keras_hub.src.models.image_segmenter_preprocessor import (
     ImageSegmenterPreprocessor,
 )
 from keras_hub.src.models.sam.sam_backbone import SAMBackbone
 from keras_hub.src.models.sam.sam_image_converter import SAMImageConverter
+from keras_hub.src.utils.tensor_utils import preprocessing_function
 
 
+@keras.saving.register_keras_serializable(package="keras_hub")
 @keras_hub_export("keras_hub.models.SamImageSegmenterPreprocessor")
```

Comment:
Should we register this also as `SAMImageSegmenter`? or keep as `SamImageSegmenter`?

Reply:
Done